tests/manage/test_create_pod_attaching_multiple_PVC_start_IO.py

```diff
@@ -0,0 +1,180 @@
+    RBD_STORAGE_CLASS_OBJ.delete()
+    RBD_POOL_SECRET_OBJ.delete()
+    RBD_POOL.delete()
+
+
+def create_pvc(pvc_name_prefix, pvc_count):
+    """
+    Function to create multiple PVC based on count and name
+
+    Args:
+        pvc_name_prefix (str): User defined pvc prefix name
+        pvc_count (int): Specify number of pvc count to be create
+
+    Returns:
+        pvc_list (list): Returns the pvc created
+    """
+    pvc_list = []
+    for count in range(pvc_count):
+        name = pvc_name_prefix + str(count)
+        pvc_obj = helpers.create_pvc(sc_name=RBD_STORAGE_CLASS_OBJ.name, pvc_name=name)
+        pvc_list.append(pvc_obj.name)
+    return pvc_list
+
+
+def create_pod(pvc_list):
+    """
+    Function to create Cirros pod with list of pvc.
+
+    Args:
+        pvc_list (list): List of PVC to be mounted in the cirros pod
+
+    Returns:
+        cirros_pod_name (str): Cirros pod deployment config name
+    """
+    temp_dict1 = []
+    temp_dict2 = []
+    temp_str = ''
+    cirros_yaml_data = templating.load_yaml_to_dict(constants.CIRROS_APP_POD_YAML)
+    # Following code collects the existing cirros yaml data, removes some
+    # values which will be updated as per pvc_list argument
+    cirros_pod_name = cirros_yaml_data['metadata']['name']
+    cirros_yaml_data['metadata']['namespace'] = config.ENV_DATA['cluster_namespace']
+    temp_list1 = cirros_yaml_data['spec']['template']['spec']['containers'][0]['args']
+    temp_list1.pop(len(temp_list1) - 1)
+    temp_list2 = cirros_yaml_data['spec']['template']['spec']['containers'][0][
+        'livenessProbe'
+    ]['exec']['command']
+    temp_list2.pop(len(temp_list2) - 1)
+    # for loop which creates the values, which will be added in cirros.yaml
+    for pv in pvc_list:
+        temp_dict1.append(
+            {'name': pv, 'persistentVolumeClaim': {'claimName': pv}}
+        )
+        temp_dict2.append({'mountPath': '/mnt' + pv, 'name': pv})
+        temp_list2.append(
+            'mount | grep /mnt' + pv + ' && head -c 1024 '
+                                       '< /dev/urandom >> /mnt' + pv + '/random-data.log'
+        )
+        temp_str = temp_str + '(mount | grep /mnt' + pv + ') && (head -c ' \
+                              '1000 < /dev/urandom > /mnt' + pv + '/random-data.log) || ' \
+                              'exit 1; '
+    del cirros_yaml_data['spec']['template']['spec']['volumes']
+    del cirros_yaml_data['spec']['template']['spec']['containers'][0]['volumeMounts']
+    del cirros_yaml_data['spec']['template']['spec']['containers'][0][
+        'livenessProbe'
+    ]['exec']['command']
+    # Updating the yaml with pv_list arg values, which will be mounted
+    # in the cirros pod.
+    cirros_yaml_data['spec']['template']['spec']['volumes'] = temp_dict1
+    cirros_yaml_data['spec']['template']['spec']['containers'][0][
+        'volumeMounts'
+    ] = temp_dict2
+    temp_list1.append('while true; do ' + temp_str + ' sleep 20 ; done')
```

Comment:
@vasukulkarni, Am trying to add more PVC to the attached yaml based on test requirement, the sample yaml file has only one PVC, so part of the POD creation am editing the yaml file to add more PVC, and in each PVC IO will be started. 
In this piece of code am updating yaml line(line number 27) to add more PVC, so that IO starts and it will continue till the POD is deleted.

Reply:
got it thanks!